core/controllers/download.py

```diff
@@ -135,17 +147,17 @@ def get_download_data(
     # filter projects by itl region
     final_projects = Project.filter_projects_by_itl_regions(projects=projects, itl_regions=itl_regions)
     # get all parent programmes of projects
-    project_parent_programmes = set(project.programme for project in projects)
+    project_parent_programmes = set(project.programme for project in final_projects)
 
-    # combine filtered projects and programme child projects
+    # COMBINE filtered projects and programme child projects
     final_programmes = {*filtered_programmes, *project_parent_programmes}  # unique programmes
-    # combine filtered programmes and project parent programmes
+    # COMBINE filtered programmes and project parent programmes
     combined_projects = {*final_projects, *programme_child_projects}  # unique projects
 
-    # sort by natural keys
+    # SORT by natural keys
     sorted_programmes = sorted(final_programmes, key=lambda x: x.programme_id)
     sorted_projects = sorted(combined_projects, key=lambda x: x.project_id)
-    return sorted_programmes, programme_outcomes, sorted_projects, project_outcomes
+    return sorted_programmes, programme_outcomes, sorted_projects, project_outcomes, submission_ids
```

Comment:
I may be misunderstanding, but is this sorting serving some purpose? As the excel files are coming out unsorted (see current/recent PR to fix this) in a seemingly random order - possibly due to data being cast to a dict downstream from here (or similar)? 

In fact, I just tried removing the sorted() from the 2 preceding lines, and all local tests still pass, so is this redundant?